src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,133 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@include check-imports('ec-component-icon');
+
+@mixin ecl-checkbox(
+  $border-width: 2px,
+  $border-color-focus: $ecl-color-yellow-100,
+  $border-color: $ecl-color-grey-75,
+  $box-size: 1.25rem,
+  $box-background-disabled: $ecl-color-grey-5,
+  $box-background-checked: $ecl-color-blue-100,
+  $box-border-color-disabled: $ecl-color-grey-50,
+  $box-border-color-checked: $ecl-color-blue-100,
+  $box-border-color-hover: $ecl-color-blue-100,
+  $invalid-border-color: $ecl-color-error,
+  $invalid-border-width: 2px,
+  $label-color: $ecl-color-grey-100,
+  $label-color-disabled: $ecl-color-grey-50,
+  $outline-color: $ecl-color-yellow-100,
+  $outline-width: 2px
+) {
+  .ecl-checkbox {
+    margin: 0;
```

Comment:
```suggestion
    align-items: flex-start;
    display: flex;
    flex-direction: column;
    margin: 0;
```

Reply:
i agree, i see that we handle better the internal margin set by the helper text, this way.